ini/stm32f4.ini

```diff
@@ -255,14 +255,13 @@ board_build.encrypt = firmware.bin
 board_build.offset  = 0x10000
 build_flags         = ${stm32_variant.build_flags}
                       -DSTM32F4 -DSTM32F4xx -DTARGET_STM32F4
-                      -DDISABLE_GENERIC_SERIALUSB -DARDUINO_ARCH_STM32 -DARDUINO_LERDGE
-                      -DHAL_SRAM_MODULE_ENABLED
+                      -DDISABLE_GENERIC_SERIALUSB -DARDUINO_ARCH_STM32 -DARDUINO_LERDGE -DLERDGE_TFT35
```

Comment:
What is the purpose of `-DARDUINO_LERDGE` ?

Reply:
i didnt find a reason for now...

Reply:
If you search for -DARDUINO_ You'll see the stuff to the right it functions as a hint to the variant (which shouldn't be needed since Marlin has a variant, you're not dependent on my hacked framework). I think this can be removed.